arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1028,8 +1028,9 @@
           "description": "The SKU name of the the container registry. Required for registry creation.",
           "enum": [
             "Basic",
-            "Standard",
-            "Premium"
+            "Standard_Small",
+            "Standard_Medium",
+            "Standard_Large"
```

Comment:
This api-version is still under private preview. While this is a breaking change there will be no impact.